share/spice/coderwall/coderwall.js

```diff
@@ -58,8 +52,35 @@
             },
 
             templates: {
-                group: 'icon'
+                group: 'icon',
+                variants: {
+                    iconImage: 'large'
+                },
+                options: {
+                    chompContents: true,
```

Comment:
Looks like you figured it out?

`condense` is a block helper. It'll apply it's changes to the final value of whatever is inside it's block. It's built to prevent breaking HTML so something like this should work:

```handlebars
{{#condense 300}}
  {{parseMD description}}
{{/condesnse}}
```